config/puma.rb

```diff
@@ -1,15 +0,0 @@
-#!/usr/bin/env puma
-
-root = File.dirname(__FILE__) + '/../'
-
-directory root
-rackup root + 'config.ru'
-environment 'production'
-bind 'tcp://0.0.0.0:8080'
-daemonize unless ENV['DOCKERIZED']
```

Comment:
Puppet [monkey-patch `daemonize`][ref], so this conf cause trouble.  But it is not needed to run the application through rack so this is I think the way to go now.

[ref]:https://github.com/puppetlabs/puppet/blob/2c1ce461ab65371a78f65233d038d113bef54ae8/lib/puppet/util/monkey_patches.rb#L27-L29